packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -278,7 +278,9 @@ class _CupertinoScrollbarState extends State<CupertinoScrollbar> with TickerProv
     WidgetsBinding.instance.addPostFrameCallback((Duration duration) {
       if (widget.isAlwaysShown) {
         _fadeoutTimer?.cancel();
-        widget.controller.position.didUpdateScrollPositionBy(0);
+        if (mounted && widget.controller.hasClients) {
```

Comment:
I'm on board with the `mounted` part of this check, but here's a potential problem caused by checking for `widget.controller.hasClients`:

  1. Pass a controller to the Scrollbar, but don't pass it to a scroll view.

```dart
            child: Scrollbar(
              controller: controller,
              isAlwaysShown: _isAlwaysShown,
              child: ListView(
                //controller: controller,
                children: <Widget>[
                  // ... Lots of children
                ],
              ),
            ),
```

  2. Before scrolling at all, toggle `_isAlwaysShown` to true.
  3. The scrollbar will not appear, and no error will be logged.

Without this change, an error will be logged when _isAlwaysShown is set to true, even if some scrolling is done first, so the developer is likely to notice it and fix it.

Is there a case where the user purposely doesn't want to pass the controller to a scroll view?